lib/fs.js

```diff
@@ -75,7 +74,8 @@ const {
     ERR_FS_FILE_TOO_LARGE,
     ERR_INVALID_ARG_VALUE,
     ERR_INVALID_ARG_TYPE,
-    ERR_FEATURE_UNAVAILABLE_ON_PLATFORM
+    ERR_FEATURE_UNAVAILABLE_ON_PLATFORM,
+    ERR_OUT_OF_RANGE
```

Comment:
nit: adding a trailing comma to simplify future changes
```suggestion
    ERR_OUT_OF_RANGE,
```

Reply:
@aduh95 do you know how to commit these suggestions as valid `fixup!` commits that can be `autosquashed`?

Reply:
Just add a `fixup!` prefix to the commit message: 

```
fixup! fs: allow `position` parameter to be a `BigInt` in read and readSync
```

Reply:
@aduh95 thank you :)